CHANGES.rst

```diff
@@ -75,8 +75,8 @@ Cubeviz
 - Calling ``cubeviz.load_data(data, data_label)``, where ``data_label`` is passed in
   as second positional argument instead of keyword, is now allowed. [#1644]
 
-- A warning will be presented with a warnning when overwriting a moment map on 
-  an existing file on disk [#1683]
+- A warning will be presented when overwriting a moment map to
+  an existing file on disk. [#1683]
```

Comment:
Should this PR just be listed in the same change log entry?

```suggestion
- A warning will be presented when overwriting a moment map to
  an existing file on disk. [#1683, #1684]
```

Reply:
Ah, yes, I guess we can. Thanks!

Yes, I agree that approval from @duytnguyendtn would be very good.